content/news/XXXX-XX-XX-mixxx-2-3-0-released.md

```diff
@@ -0,0 +1,122 @@
+status: draft
+comments: yes
+
+After a long waiting time, we are proud to announce the release of Mixxx 2.3.0, [download it!]({filename}/pages/download.md#stable)
+This release brings [hotcue colors & labels]({filename}/news/2020-08-25-new-in-2-3-hotcue-colors.md), import of [Serato](2021-02-08-new-in-2-3-serato-support.md) and [Rekordbox]({filename}/news/2020-07-20-new-in-2-3-rekordbox-support.md) libraries, [more accurate key detection]({filename}/news/2021-04-08-new-in-2-3-keyfinder.md), [intro/outro sections]({filename}/news/2020-07-09-intro-outro-sections.md), a revamped LateNight skin, multithreaded analysis, [deck cloning]({filename}/news/2020-11-15-new-in-2-3-deck-clone.md), [track color support]({filename}/news/2020-10-24-new-in-2-3-track-colors.md).
+
+We also added support for recording/streaming in the [Opus codec](https://opus-codec.org/), polished the preferences (including the controller workflow), added some library improvements and [switched the CMake build system generator]({filename}/news/2020-12-14-scons-cmake-migration.md).
+
+2.3.0 also adds out-of-the-box support for the [Pioneer DDJ-200](https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_200.html) and [DDJ-400](https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_400.html), the [Native Instruments Traktor Kontrol S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html), the [Hercules DJControl Inpulse 200](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_200.html) and [Jogvision](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_jogvision.html), the [Roland DJ-505](https://manual.mixxx.org/2.3/en/hardware/controllers/roland_dj_505.html), the [Behringer B-Control BCR2000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_bcr2000.html) and [DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [ION Discover DJ Pro](https://manual.mixxx.org/2.3/en/hardware/controllers/ion_discover_dj_pro.html) and the [Numark iDJ Live II](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_idj_live_ii.html). A few existing controller mappings have received fixes and new features, too.
+
+For the full list of changes, have a look at the [changelog](https://manual.mixxx.org/2.3/en/chapters/appendix/version_history.html) and the [2.3.0 milestone on Launchpad](https://launchpad.net/mixxx/+milestone/2.3.0).
+
+This has been a rather big release, with lots of useful changes, bug fixes and improvements.
+All in all, a total of 7183 changes changed over 1 million lines of code were changed since the [2.2.4 release]({filename}/news/2020-06-11-Mixxx-2-2-4-released.md)!
+
+**TODO:** Update the numbers above using `git shortlog -sn release-2.2.4..upstream/2.3 | wc -l`, `git log --oneline --no-merges release-2.2.4..upstream/2.3 -- . ':!lib' | wc -l` and `git diff --shortstat release-2.2.4..upstream/2.3 -- . ':!lib`.
```

Comment:
does markdown support comments, so that this will be guaranteed not to appear in the final version?

Reply:
This is not a comment, it's just a reminder to do this after tagging and before merge, and then remove that comment.

Reply:
yeah I know, I have just seen enough blog posts go live with "TO DO" "TKTKTKTKTK" "USE REAL DATA" that I prefer never to have it in there at all. don't worry about it. (https://stackoverflow.com/questions/4823468/comments-in-markdown)